app/components/initEndpoint/initEndpoint.html

```diff
@@ -53,12 +53,8 @@
               <div class="form-group">
                 <div class="col-sm-12">
                   <span class="small">
-                    <p class="text-muted">
-                      <i class="fa fa-exclamation-triangle orange-icon" aria-hidden="true" style="margin-right: 2px;"></i>
-                      This feature is not yet available for <u>native Docker Windows containers</u>.
-                    </p>
                     <p class="text-primary">
-                      Please ensure that you have started the Portainer container with the following Docker flag <code>-v "/var/run/docker.sock:/var/run/docker.sock"</code> in order to connect to the local Docker environment.
+                      Please ensure that you have started the Portainer container with the following Docker flag <code>-v /var/run/docker.sock:/var/run/docker.sock</code> (MacOS/Linux) or <code>-v \\.\pipe\docker_engine:\\.\pipe\docker_engine</code> (Native Windows containers) in order to connect to the local Docker environment.
```

Comment:
I suggest to remove MacOS and instead only mention Linux containers. 
I'm working from a Mac to start Portainer on a Windows Docker host as well. 
This reminds me of LCOW, let's see what combinations we can use there? 🤓

Reply:
For now, I'd rather not see the information message for the Windows platform. As this feature is not available yet for Windows container, it may confuse our current Windows users.

I'd go for merging the support of the named pipe, without really being able to use it before the ability to mount the named pipe is released (no changes on the front-end for now).